docs/app/Examples/modules/Popup/Triggers/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+
+export default function () {
+  return (
```

Comment:
All files' default export should match the filename.  Since index.js files don't make sense in this pattern, we're using the "Component + Directory + Examples".  Also note the const vs function, let's update to use:

```js
const PopupTriggersExamples = () => (
  // ...
)
export default PopupTriggersExamples
```